src/options.h

```diff
@@ -601,11 +601,11 @@ sp_before_class_colon;
 
 // Add or remove space after class constructor ':'.
 extern Option<iarf_e>
-sp_after_constr_colon;
+sp_after_constr_colon; // = IARF_ADD
```

Comment:
NO!
we cannot change a default value, which is already in use by many users.

Reply:
The default behavior doesn't change. Before this patch, sp_after_constr_colon=ignore and sp_after_constr_colon=add did the same thing (unless indent_ctor_init_leading is changed from its default value of 2). After this patch, setting sp_after_constr_colon=ignore works as expected and does not add any spaces. Therefore the default value of sp_after_constr_colon has to change to preserve the default behavior that users expect.